packages/opentelemetry/lib/plugin.js

```diff
@@ -0,0 +1,45 @@
+import BasePlugin from '@appium/base-plugin';
+import { tracerProviderInstance } from './tracing/tracerProvider';
+
+const STATUS_MESSAGE = {
+  ACTIVE: 'active',
+  INACTIVE: 'inactive',
+  SUCCESS: 'success',
+  FAILURE: 'failure',
+  INVALID_EXPORTER_ERROR_MESSAGE: 'invalid exporter object'
+};
+
+export default class OpentelemetryPlugin extends BasePlugin {
+  constructor (pluginName, opts = {}) {
+    super(pluginName, opts);
+  }
+
+  static async updateServer (expressApp/*, httpServer*/) { // eslint-disable-line require-await
+    expressApp.post('/opentelemetry/config', OpentelemetryPlugin.setOpentelemetryConfig);
+    expressApp.get('/opentelemetry/config', OpentelemetryPlugin.getOpentelemetryConfig);
+    expressApp.get('/opentelemetry/status', OpentelemetryPlugin.getStatus);
+  }
+
+  static getOpentelemetryConfig (req, res) {
```

Comment:
I'd add a `handle` prefix to these method names, since they are endpoint handlers, for example `handleSetOpentelemetryConfig`

Reply:
updated